build/stf-run-ci/tasks/main.yml

```diff
@@ -75,6 +78,17 @@
     when:
     - __loki_skip_tls_verify | bool
 
+  - name: Replace loki-operator golang base image
+    replace:
+      path: "{{ playbook_dir }}/working/loki-operator/Dockerfile"
+      regexp: "golang:1.16"
+      replace: "{{ __golang_image_path }}"
+
+  - name: Replace Loki image
+    replace:
+      path: "{{ playbook_dir }}/working/loki-operator/internal/manifests/var.go"
+      regexp: "docker.io/grafana/loki:\\d\\.\\d\\.\\d"
```

Comment:
The regexp should probably be wider/simpler, like `docker.io/grafana/loki:.*` in case the tag style changes.